smartdashboard/schemas/shard.py

```diff
@@ -33,3 +33,6 @@ class Shard(EntityWithNameTelemetryMetaDataErrOut):
     hostname: str
     port: int
     conf_file: t.Optional[str]
+    memory_file: str
+    client_file: str
+    client_count_file: str
```

Comment:
Are these paths optional? If specific collectors are disabled (or the default behavior w/manifest only), I think these may not be populated?

Reply:
Make these `= ""`